### DISCUSSION TITLE
Actions deploy.yml error

### DISCUSSION THREAD

**Discussion by divoritchie** [vuejs/vitepress#2055] [Q&A]
### Describe the bug

Run actions/setup-node@v3

### Reproduction

Reproduction

### Expected behavior

pass

### System Info

```shell
Error: Dependencies lock file is not found in /home/runner/work/ritchie2000.github.io/ritchie2000.github.io. Supported file patterns: yarn.lock
```

### Validations

- [X] Follow our [Code of Conduct](https://vuejs.org/about/coc.html)
- [X] Read the [docs](https://vitepress.vuejs.org).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.

**Comment by brc-dd** [comment]
You forgot to upload the lock file?

**Reply by lazarusA** [reply]
How do one gets this file? Doing the [initial setup](https://vitepress.dev/guide/getting-started#setup-wizard) only one `json` file is created. And then when trying to use the [deploy](https://vitepress.dev/guide/deploy#github-pages) instructions the above error dependency is shown.
